ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -257,65 +192,50 @@ Now press **"CTRL+O"**  and press enter to save the file, afterwards exit it by
 
 Now test the Connection to the server and type following:
 
-```
-pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY
-```
-If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **"CTRL+C"**.
+- **`# pyhoca-cli --new  --server 92.246.17.152  --port 58005 --user testuser --pass testuser  --command UNITY`**
+
+If you connect then an x2go session will start and it will show you a new desktop. Exit the desktop by logging out and the press **`"CTRL+C"`**.
 
 Now edit the environment file, open the file by typing:
 
-```
-nano /etc/environment
-```
+- **`# nano /etc/environment`**
 
 In the environment press down arrow in order to go to a new line and insert following line:
 
-```
-SERVER_ROOT=http://92.246.17.152:8080/uccs/inifile/
-```
+- **`# SERVER_ROOT=http://92.246.17.152:8080/uccs/inifile/`**
 
 Now press **"CTRL+O"** and press enter to save the file, afterwards exit it by pressing **"CTRL+X"**.
 
 Now run following commands, though NOTE: username and password are both testuser, also when typing password and username do not include **<>**.
 
-```
-export SERVER_ROOT
-```
+- **`# export SERVER_ROOT`**
 
 Type exit and press enter this will put you back to the very beginning, log back in as an Admin/Superuser/ROOT by typing following:
 
-```
-sudo su
-```
+- **`$ sudo -i`**
+
 type your password as you did in the beginning.
 
 Now type following to get information about remote desktop servers:
 
-```
-echo <password> | thin-client-config-agent <username>
-```
+- **`# echo <password> | thin-client-config-agent <username>`**
 
 In your case it could look like:
 
-```
-echo testuser | thin-client-config-agent testuser
-```
+- **`# echo testuser | thin-client-config-agent testuser`**
 
 In order to connect to a remote server you have to tell it which IP to ping, therefore open the remote-logon-service file by typing following:
 
-```
-nano /etc/remote-logon-service.conf
-```
+- **`# nano /etc/remote-logon-service.conf`**
 
 Change the line 
 `URI=http://localhost:8080/uccs/inifile/` to:
 
-```
-URI=http://92.246.17.152:8080/uccs/inifile/
-```
+- **` URI=http://92.246.17.152:8080/uccs/inifile/**
+
 **"CTRL+O"**  and press enter to save the file, afterwards exit it by pressing **"CTRL+X"**.
 
-Type **"exit"** in order to logout as Admin, type **"exit"** again so it will close the Terminal. Restart the system and you will see Arctica Greeter as your login screen.
+Type **`exit`** in order to logout as Admin, type **`exit`** again so it will close the Terminal. Restart the system and you will see Arctica Greeter as your login screen.
```

Comment:
WTH, I must have done something wrong with git again >:( .........
I have the absolute corrected version on my notebook right here in front of me